src/Parser.js

```diff
@@ -554,10 +557,10 @@ export default class Parser {
      */
     parseArguments(
         func: string,   // Should look like "\name" or "\begin{name}".
-        funcData: FunctionSpec | EnvSpec,
+        funcData: FunctionSpec<*> | EnvSpec,
```

Comment:
Will `EnvSpec` need to be parameterized eventually as well?

Reply:
Unsure. I'll see once I try to port buildHTML over.

Reply:
The answer is yes. This is small enough that I can do it fully (both the `hander` and the `htmlBuilder` part) in one fell swoop. It requires a helper from the next PR, so I'm going to keep it as part of the next PR for now.